src/ResourceManager/SignalR/Commands.SignalR/Cmdlets/GetAzureRmSignalR.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.Azure.Commands.SignalR.Cmdlets
     [OutputType(typeof(PSSignalRResource))]
     public class GetAzureRmSignalR : SignalRCmdletBase, IWithResourceId
     {
-        [Parameter(
+        [Parameter(Position = 0,
```

Comment:
I don't think we should use Position in a non-mandatory parameter.

Reply:
reverted and updated the position for other mandatory parameters after discussion with Sergey. Docs were updated accordingly.